apps/docs-app/package.json

```diff
@@ -11,16 +11,30 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.1",
-    "@docusaurus/preset-classic": "2.0.1",
+    "@docusaurus/core": "2.2.0",
+    "@docusaurus/preset-classic": "2.2.0",
     "@mdx-js/react": "1.6.22",
     "clsx": "1.2.1",
     "prism-react-renderer": "1.3.5",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
```

Comment:
quick question but why do you want to use React 18 when Docusauurs does not provide yet official support?

You could as well stay on React 17 and use Docusaurus 2.3 (the recommended setup)

Reply:
Thanks, I didn't know it wasn't fully supported yet. I'm fine with going back to the recommended setup for now